functions/billing/walkthroughtutorial.md

```diff
@@ -0,0 +1,154 @@
+# Automating cost controls by capping billing
```

Comment:
Nit: can we name this file `tutorial.md`? (A one-word name is probably cleaner.)

Reply:
Thanks for the questions, @ace-n.
1a) No, this will only be in Python.
1b) No, there aren't requests for multiple languages (at least now). If that changes when it goes public, we'll address it then.
2) I can see where that might seem reasonable, but we chose Neos and GitHub because @hujessica is interning with us this summer.

Regarding @grant's suggestion, I agree that it can live in a `billing/` folder in this repo. We anticipate/hope to create more Neos-style tutorials in the future. And if we're using a `billing/` folder, then I think the name should remain as-is, capbillingtutorial.md.

Reply:
Got it - thanks for the clarification!

A few further questions:
1) I'm fine with longer names, but can we break them up a bit (e.g. hyphenate them)?
2) Is this repo the best place to put Neos tutorials? (i.e. would it be better to create a separate GitHub repo dedicated to them, assuming they're _supposed_ to live on a public GitHub a la [our community tutorials site](https://cloud.google.com/community)?)
3) I like @grant's suggestion of using a `billing` folder. If we merge these samples into this repo, we should identify an owner and document that in the repo-root `CODEOWNERS` file.

(Note to self: if we move this to a separate folder, we'll have to do the same for Node.)

Reply:
ALso - N.B. the Node implementation is [here](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/functions/billing/index.js).

(I presume the reason for **not** reusing that code is that external developers are more familiar with Python.)